### ISSUE TITLE
Types field is missing

### ISSUE THREAD

**Issue by DarkerTV** [bugsnag/bugsnag-js#232]
Currently package.json file at master have a `types` field pointing to `src/bugsnag.d.ts`. But if you install the package running `npm install --save bugsnag-js` and open the package.json file at `node_modules/bugsnag-js` you see something like this.

```json
{
  "_args": [
    [
      {
        "raw": "bugsnag-js",
        "scope": null,
        "escapedName": "bugsnag-js",
        "name": "bugsnag-js",
        "rawSpec": "",
        "spec": "latest",
        "type": "tag"
      },
      "C:\\Users\\tarde\\Desktop\\test"
    ]
  ],
  "_from": "bugsnag-js@latest",
  "_id": "bugsnag-js@3.1.0",
  "_inCache": true,
  "_location": "/bugsnag-js",
  "_nodeVersion": "6.4.0",
  "_npmOperationalInternal": {
    "host": "packages-18-east.internal.npmjs.com",
    "tmp": "tmp/bugsnag-js-3.1.0.tgz_1487626738507_0.82040149345994"
  },
  "_npmUser": {
    "name": "eanakashima",
    "email": "e.a.nakashima@gmail.com"
  },
  "_npmVersion": "3.10.3",
  "_phantomChildren": {},
  "_requested": {
    "raw": "bugsnag-js",
    "scope": null,
    "escapedName": "bugsnag-js",
    "name": "bugsnag-js",
    "rawSpec": "",
    "spec": "latest",
    "type": "tag"
  },
  "_requiredBy": [
    "#USER",
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/bugsnag-js/-/bugsnag-js-3.1.0.tgz",
  "_shasum": "231105e590895282f730a3e31ec83801d48bee5c",
  "_shrinkwrap": null,
  "_spec": "bugsnag-js",
  "_where": "C:\\Users\\tarde\\Desktop\\test",
  "bugs": {
    "url": "https://github.com/bugsnag/bugsnag-js/issues"
  },
  "dependencies": {},
  "description": "[![Documentation](https://img.shields.io/badge/docs-latest-blue.svg)](https://docs.bugsnag.com/platforms/browsers) [![Build status](https://travis-ci.org/bugsnag/bugsnag-js.svg?branch=master)](https://travis-ci.org/bugsnag/bugsnag-js)",
  "devDependencies": {
    "eslint": "^3.4.0",
    "grunt": "~0.4.2",
    "grunt-bumpx": "~0.1.0",
    "grunt-contrib-watch": "~0.5.2",
    "grunt-docco": "~0.4.0",
    "grunt-invalidate-cloudfront": "~0.1.4",
    "grunt-regex-replace": "~0.2.5",
    "grunt-s3": "git://github.com/pifantastic/grunt-s3.git",
    "gruntify-eslint": "^1.3.0",
    "karma": "~1.3.0",
    "karma-mocha-ie-legacy": "file:./test/karma-mocha",
    "karma-phantomjs-launcher": "^1.0.1",
    "karma-sauce-launcher": "^1.0.0",
    "mocha": "~1.15.1",
    "uglifyjs": "latest"
  },
  "directories": {},
  "dist": {
    "shasum": "231105e590895282f730a3e31ec83801d48bee5c",
    "tarball": "https://registry.npmjs.org/bugsnag-js/-/bugsnag-js-3.1.0.tgz"
  },
  "gitHead": "a68d6f05fb4ce04ed5cc3e9974815904143cf714",
  "homepage": "https://github.com/bugsnag/bugsnag-js#readme",
  "license": "MIT",
  "main": "src/bugsnag.js",
  "maintainers": [
    {
      "name": "eanakashima",
      "email": "e.a.nakashima@gmail.com"
    },
    {
      "name": "kattrali",
      "email": "iskanamagus@gmail.com"
    },
    {
      "name": "snmaynard",
      "email": "simon@snmaynard.com"
    },
    {
      "name": "wordofchristian",
      "email": "christian@cswebartisan.com"
    }
  ],
  "name": "bugsnag-js",
  "optionalDependencies": {},
  "readme": "ERROR: No README data found!",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/bugsnag/bugsnag-js.git"
  },
  "scripts": {
    "lint": "grunt eslint",
    "test": "karma start --single-run",
    "test:quick": "karma start --single-run --browsers PhantomJS",
    "test:watch": "karma start --browsers PhantomJS"
  },
  "version": "3.1.0"
}
```

The field `"types": "src/bugsnag.d.ts",` is missing.

**Comment by kattrali** [comment]
Thanks for the report, @DarkerTV. There hasn't been anything to release since adding the typescript definitions to the package.json in https://github.com/bugsnag/bugsnag-js/commit/e66ed2e5942256b46b77993221bfac9e0ca6efeb. It might make sense to bump a release for it though.

**Comment by kattrali** [comment]
Fixed in 3.2.0
